### ISSUE TITLE
encoding, contentType: image/jpeg not validated

### ISSUE THREAD

**Issue by tamis-laan** [cdimascio/express-openapi-validator#693]
I have a request with the following body:

``` yaml
     requestBody:
       content:
         multipart/form-data:
           schema:
             type: object
             properties:
               avatar:
                 type: string
                 format: binary
           encoding:
             avatar:
               contentType: image/jpeg
```

However I can just upload a `image/png` and express-openapi-valdator doesn't care :-/

**Comment by sghoe** [comment]
same is for mp4, there is a part missing
@tamis-laan how did you manage to?

**Comment by tamis-laan** [comment]
@sghoe Nope, I postponed this for other work. Planning to fix this by manually checking the media type for now. I remember there is a package called mime for nodejs that you can use: 
https://www.npmjs.com/package/mime 
But ugly solution...

**Comment by mzbik** [comment]
I have a similar issue 
`      responses:
        400:
          description: Failure, see body
          content:
            application/json:
              schema:
                type: string
        200:
          description: Successfully returning document
          content:
            application/pdf:
              schema:
                type: string
                format: binary
`
When building the response validator (openapi.response.validators/buildValidators), the 200 response fails canValidateContentType (line ~177) meaning that in line ~204 "types" is empty and no response validator for 200 is generated.  The behavior is then that 200 returns are failed by the validator with "missing schema".

A quick fix would be to add 'pdf' in the test at line '251'.  A longer-term fix would be to allow adding of user-generated validators per-content-type.
